SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialLatencyTracer.h"
+
+#include "Async/Async.h"
+#include "Engine/World.h"
+#include "EngineClasses/SpatialGameInstance.h"
+#include "Interop/Connection/OutgoingMessages.h"
+#include "Utils/SchemaUtils.h"
+
+#include <sstream>
+
+DEFINE_LOG_CATEGORY(LogSpatialLatencyTracing);
+
+// Stream for piping trace lib output to UE output
+class UEStream : public std::stringbuf
+{
+	int sync()
+	{
+		UE_LOG(LogSpatialLatencyTracing, Verbose, TEXT("%s"), *FString(str().c_str()));
+		str("");
+		return std::stringbuf::sync();
```

Comment:
I don't think this function is defined other than as a call to `std::basic_streambuf::sync()` which is also does nothing (other than return 0).

Reply:
Can't hurt right?

Reply:
It would be clearer possibly to return 0 directly as that indicates the that is has been successful.